Branch-SDK/src/main/java/io/branch/referral/validators/AlternateDomainsCheck.java

```diff
@@ -0,0 +1,47 @@
+package io.branch.referral.validators;
+
+import android.content.Context;
+import android.text.TextUtils;
+
+import org.json.JSONObject;
+
+public class AlternateDomainsCheck extends IntegrationValidatorCheck {
+    String name = "Alt Domains";
+    String errorMessage = "Could not find intent filter to support alternate link domain. Please add intent filter for handling alternate link domain in your Android Manifest file";
+    String moreInfoLink = "<a href=\"https://help.branch.io/developers-hub/docs/android-basic-integration#:~:text=The%20%2Dalternate.app.link%20domain%20associated%20with%20your%20app\">More info</a>";
```

Comment:
Can we have these links as their own constants? Documentation is subject to change, would be helpful to have these values centralized.

Reply:
Good point, updated!